src/mixins/I13nUtils.js

```diff
@@ -86,9 +94,10 @@ var I13nUtils = {
      * @return {Object} parent i13n node
      */
     _getParentI13nNode: function () {
-        // https://twitter.com/andreypopp/status/578974316483608576, get the context from parent context
+        var reactI13n = this._getReactI13n();
         var context = this.context;
-        return (context && context.i13n && context.i13n.parentI13nNode) || this._getReactI13n().getRootI13nNode();
+        return (context && context.i13n && context.i13n.parentI13nNode) || 
+            (reactI13n && reactI13n.getRootI13nNode());
```

Comment:
What are the use cases of getting undefined?  The returned value is exposed thru public API as child context.  If there are real scenarios, we should document how people should treat the `undefined` return value.